docs/java-rest/high-level/indices/refresh.asciidoc

```diff
@@ -0,0 +1,75 @@
+[[java-rest-high-refresh]]
+=== Refresh API
+
+[[java-rest-high-refresh-request]]
+==== Refresh Request
+
+A `RefreshRequest` can be applied to one or more indices, or even on `_all` the indices:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests}/IndicesClientDocumentationIT.java[refresh-request]
+--------------------------------------------------
+<1> Refresh one index
+<2> Refresh multiple indices
+<3> Refresh all the indices
+
```

Comment:
could you please the optional arguments section like we have in other places and mention the indices options (ignore_unavailable, allow_no_indices and expand_wildcards)?